README.md

```diff
@@ -29,11 +29,11 @@ for up to date installation instructions.
 ```swift
 import Swish
 
-func baseRequest(url url: String, method: RequestMethod) {
+func baseRequest(url url: String, method: RequestMethod) -> NSURLRequest {
   let url = NSURL(string: url)!
   let request = NSMutableURLRequest(URL: url)
 
-  request.HTTPMethod = method.rawValue
+  return request.HTTPMethod = method.rawValue
```

Comment:
yeah, I don't think this will work. There should be an added `return request` line instead.